cdhweb/pages/management/commands/exodus.py

```diff
@@ -243,7 +243,7 @@ def handle(self, *args, **options):
 
             # project list pages become link pages
             project_pages = MezzaninePage.objects \
-                .filter(Q(slug__startswith="projects")).order_by('-slug')
+                .filter(Q(slug__startswith="projects")).order_by('_order')
```

Comment:
does this need `Q`?

Reply:
No! Bet it was copied from a different one, that does